src/Attributes/Attribute.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * This file is part of riesenia/routing package.
+ *
+ * (c) RIESENIA.com
+ */
+declare(strict_types=1);
+
+namespace Riesenia\Routing\Attributes;
+
+abstract class Attribute
+{
+    protected string $name;
+
+    public function __construct(
```

Comment:
i see no logic in defining the constructor and initialize here.

Reply:
i'm happy you resolved this, but i'd better get an explanation

Reply:
so what i had in mind when extending the resource/route we wouldn't need to use the constructor and defining all the params 
we would use the initalise method and use the default values alreay in parent class and change the ones we want. 

I saw it's cleaner with the initialise.